ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,57 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
+  // TODO: remove exporationExprots
+  const { download, deleteDownloadModel } = props;
+
+  let expireExportsString = '';
```

Comment:
I would extract this to a function called `getDownloadDisplayName` to more cleanly contain this logic and avoid the `let`.

Reply:
https://github.com/LearningLocker/learninglocker/commit/80fb50f846b2ce69ca86bdcbe27948c62276945d fixed